bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: Added support for KFM Billing Models to QUOTA-LIST

## Description
Added support for KFM billing model to quota-list so that different billing model are supported for the same instance type (standard, eval, marketplace, ...)

## Verification Steps

1) Run KFM locally with QUOTA-LIST and the specified configuration file. Try to create kafkas and check that it gives an error after reaching the quota.
2) Try to create a kafka with a user not listed in the QUOTA-LIST (set any_user to false) and ensure it allows you to create only one developer instance.
3) Run KFM locally with  AMS and check that it still works as before

**Example configuration file**
https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/bde4796f5700b99671e532ac008d4c8be039c1fd/config/quota-management-list-configuration.yaml

## Checklist (Definition of Done)
